sdk/logical/rotation_job.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package logical
```

Comment:
Common helpers to create Rotation Jobs. Will be used by all plugins, hence added to CE to avoid SDK splitting

Reply:
I am wondering if we considered adding this as a top-level packange in `sdk/`? Something like `sdk/rotation`.

The developer experience with using the `logical` package is less than ideal. The name `sdk/logical` is quite mysterious whereas something like `sdk/rotation` is much clearer.

Reply:
Moved to a top-level `sdk/rotation` package